ui/app/pages/swap/swap-content/swap-amount-row/swap-amount-row.component.js

```diff
@@ -105,40 +104,22 @@ export default class SwapAmountRow extends Component {
     this.updateAmount(newAmount)
   }
 
-  renderInput () {
-    const { amount, inError, swapFromToken } = this.props
+  render () {
+    const { amount, swapFromToken } = this.props
 
     return swapFromToken ?
       (
         <UserPreferencedTokenInput
-          error={inError}
           onChange={this.handleChange}
           token={swapFromToken}
           value={amount}
         />
       )
       : (
         <UserPreferencedCurrencyInput
-          error={inError}
           onChange={this.handleChange}
           value={amount}
         />
       )
   }
-
-  render () {
-    const { gasTotal, inError } = this.props
-
-    return (
-      <SwapRowWrapper
-        label={`${this.context.t('amount')}:`}
-        showError={inError}
-        errorType="amount"
-      >
-        {gasTotal && <AmountMaxButton inError={inError} />}
```

Comment:
**Note to reviewer:**

I'll probably add the "Max" feature back in a later PR.